src/components/Inputs/CustomLookup.js

```diff
@@ -265,12 +267,16 @@ const CustomLookup = ({
             placeholder={secondFieldLabel == '' ? displayField.toUpperCase() : secondFieldLabel.toUpperCase()}
             value={secondValue ? secondValue : ''}
             required={isRequired}
-            disabled={disabled}
+            onChange={e => {
+              if (onSecondValueChange && secondFieldName) {
+                onSecondValueChange(secondFieldName, e.target.value)
+              }
+            }}
             InputProps={{
               inputProps: {
-                tabIndex: -1 // Prevent focus on the input field
+                tabIndex: _readOnly ? -1 : 0 // Prevent focus on the input field
```

Comment:
Are you certain? Since _readOnly makes two fields in the lookup readOnly, it's possible that only the second field is readOnly in other case.